### ISSUE TITLE
Different pot. Energy 

### ISSUE THREAD

**Issue by uarif** [ubsuny/23-Homework7G1#50]
Hello, I got different energy in my case than the paper, what could be the reason? Could it be the reason that may be our a,b, etc, values are different? What you think? Any help would be appreciated.

**Comment by yasmensarhan27** [comment]
how much did you get ??
if -28 or near so you are fine because the value in the paper is for potential energy per ion pair

**Comment by pratibha77118** [comment]
Yes exactly the value we get is total potential energy after optimization but the value given in paper is the value of binding energy per ion.

**Comment by uarif** [comment]
okay thank you
